corehq/apps/reports/models.py

```diff
@@ -561,3 +562,19 @@ def delete_user(self, id):
             {}
         )
         return True
+
+
+class QueryStringHash(models.Model):
```

Comment:
This should have a `domain` field as well.

Reply:
yeah I think on the safe side we should have a domain here on the very rare event that there might be a collision between two domain's hashes that map to different strings.